src/components/dev/env-panel/EnvPanel.tsx

```diff
@@ -0,0 +1,123 @@
+import { useEffect, useMemo, useState } from "react";
+import { useLocation } from "~wallets/router/router.utils";
+import { clsx } from "clsx";
+
+import styles from "./EnvPanel.module.scss";
+import { EMBEDDED_SERVER_BASE_URL } from "~utils/embedded/iframe.utils";
+
+export interface EnvPanelProps {}
+
+type EnvStatus = "ok" | "err" | "?";
+
+type EnvType = "local" | "preview" | "development" | "production" | "unknown";
+
+function getEnvColor(envType: EnvType) {
+  if (envType === "local") return "grey";
+  if (envType === "preview") return "yellow";
+  if (envType === "development") return "green";
+  if (envType === "production") return "red";
+  return "transparent";
+}
+
+export function EnvPanel({}: EnvPanelProps) {
+  const { location } = useLocation();
+  const [isHidden, setIsHidden] = useState(process.env.NODE_ENV !== "development");
+
+  useEffect(() => {
+    (window as any).showEnvPanel = () => {
+      setIsHidden(false);
+    };
+
+    (window as any).hideEnvPanel = () => {
+      setIsHidden(true);
+    };
+  }, []);
+
+  const { envStatus, envStatusColor, clientEnv, clientEnvColor, authEnv, authEnvColor, serverEnv, serverEnvColor } =
+    useMemo(() => {
+      let envStatus: EnvStatus = "?";
+      let clientEnv: EnvType = "unknown";
+      let authEnv: EnvType = "unknown";
+      let serverEnv: EnvType = "unknown";
+
+      const { hostname } = window.location;
+
+      if (hostname === "localhost") {
+        clientEnv = "local";
+      } else if (/^wander-embed-([a-z0-9]+)-community-labs.vercel.app$/.test(hostname)) {
+        clientEnv = "preview";
+      } else if (hostname === "connect-dev.wander.app") {
+        clientEnv = "development";
+      } else if (hostname === "connect.wander.app") {
+        clientEnv = "production";
+      }
+
+      if (import.meta.env?.VITE_SUPABASE_URL === "https://pboorlggoqpyiucxmneq.supabase.co") {
+        authEnv = "development";
+      } else if (import.meta.env?.VITE_SUPABASE_URL === "https://uqetkgfbdqrdtixnkgdm.supabase.co") {
+        authEnv = "production";
+      }
+
+      if (EMBEDDED_SERVER_BASE_URL.includes("://localhost:")) {
+        serverEnv = "local";
+      } else if (/^embed-([a-z0-9]+)-community-labs.vercel.app$/.test(hostname)) {
```

Comment:
I think this should be updated to this to match the preview url.
```suggestion
      } else if (/embed-([a-z0-9]+)-community-labs.vercel.app$/.test(EMBEDDED_SERVER_BASE_URL)) {
```